pkg/cnab/config-adapter/adapter.go

```diff
@@ -401,10 +406,9 @@ func (c *ManifestConverter) generateBundleImages() map[string]bundle.Image {
 	return images
 }
 
-func (c *ManifestConverter) generateDependencies() *cnab.Dependencies {
-
+func (c *ManifestConverter) generateDependencies() (*cnab.Dependencies, error) {
```

Comment:
Maybe I'm missing something, I can't find where this function is returning an error?

Reply:
Sorry the next PR after this needed to be able to return an error. Are you okay with me keeping that in place here so that I don't need to redo both PRs to shuffle when we introduce that change?